includes/embeds/class-amp-tiktok-embed-handler.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Class AMP_Tiktok_Embed_Handler
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_Tiktok_Embed_Handler
+ */
+class AMP_Tiktok_Embed_Handler extends AMP_Base_Embed_Handler {
+
+	/**
+	 * Registers embed.
+	 */
+	public function register_embed() {
+		add_filter( 'embed_oembed_html', [ $this, 'filter_embed_oembed_html' ], 10, 3 );
```

Comment:
Note that the raw embeds also need to be handled, that is the `blockquote` that someone pastes into a Custom HTML block for example. See `\AMP_Twitter_Embed_Handler::sanitize_raw_embeds()`. The Twitter embed handler doesn't filter `embed_oembed_html` at all. I recall discussing this with you in December, that we can just rely on `sanitize_raw_embeds` alone.

Reply:
Embeds are now being handled via `sanitize_raw_embeds`; see https://github.com/ampproject/amp-wp/pull/4357/commits/e8fa07baddd451272c0a9117c47e31ca3518d857.